### PR TITLE
Remove empty files

### PR DESCRIPTION
**Pull request by Kegsay**
As discussed previously, empty files make me a sad panda for a number of reasons:
 - Needless boilerplate when writing React classes if we are supposed to add them and have them blank.
 - Needless confusion when you're debugging problems and continually open blank files expecting there to be code there (compounded by the fact that we add block headers so the size isn't even 0 bytes!!)

If we need to add a controller file for a view, we can just add it when we need it.

Related: https://github.com/matrix-org/matrix-react-sdk/pull/22

### CODE REVIEW

**Comment by ara4n** [comment]
this looks very fine by me - i've found it a tedious and confusing mess to chase dead ends too.

File: src/skins/vector/views/pages/MatrixChat.js [hunk 1]
```diff
@@ -40,6 +39,23 @@ module.exports = React.createClass({
...
+            ContextualMenu.createMenu(MemberInfo, {
+                member: member,
+                right: window.innerWidth - e.pageX,
+                top: e.pageY,
+                onFinished: function() {
+                    console.log("^_^ All done!");
+                }
+            });
+        };
     },
```
Comment (dbkr) [inline src/skins/vector/views/pages/MatrixChat.js]: Is this change related?

File: src/skins/vector/views/atoms/MessageTimestamp.js [hunk 2]
```diff
@@ -18,14 +18,11 @@ limitations under the License.
 
 var React = require('react');
 
-var MessageTimestampController = require('matrix-react-sdk/lib/controllers/atoms/MessageTimestamp')
```
Comment (dbkr) [inline src/skins/vector/views/atoms/MessageTimestamp.js]: I only see ImageViewController being deleted? Is this just github being dumb?

Comment (dbkr) [inline src/skins/vector/views/atoms/MessageTimestamp.js]: Oh, no - those and in react-sdk, yes, of course.

File: src/skins/vector/views/pages/MatrixChat.js (hunk 1 above)
Comment (Kegsay) [inline src/skins/vector/views/pages/MatrixChat.js]: Oops! will fix, sorry.

**Comment by dbkr** [comment]
LGTM

File: src/skins/vector/views/pages/MatrixChat.js (hunk 1 above)
Comment (Kegsay) [inline src/skins/vector/views/pages/MatrixChat.js]: Fixed.
